httpserver/lmq_server.h

```diff
@@ -64,15 +67,15 @@ class OxenmqServer {
     // request name and a callback) are forwarded as `omq.request(connid, ...)`.
     template <typename... Args>
     void oxend_request(Args&&... args) {
-        if (!oxend_conn_) { std::abort(); }
+        assert(oxend_conn_);

```

Comment:
would it be better to throw on no connection instead of an assert?

Reply:
No, this *should* always be set (if it isn't, something is broken in the application setup code).